lr-cli/cortx_setup/commands/storageset/config/durability.py

```diff
@@ -64,52 +73,71 @@ class DurabilityConfig(Command):
 
     def run(self, **kwargs):
         try:
-            index = 'storage_durability_index'
+            confstore_file_index = 'storage_durability_index'
             storage_set_name = kwargs['storage_set_name']
             durability_type = kwargs['durability_type']
             cluster_id = get_cluster_id()
+            durability_dict = {
+                'data': str(
+                    kwargs['data']), 'parity': str(
+                    kwargs['parity']), 'spare': str(
+                    kwargs['spare'])}
 
             self.load_conf_store(
-                index,
+                confstore_file_index,
                 f'json://{CONFSTORE_CLUSTER_FILE}'
             )
 
-            ss_name = Conf.get(index,
-                               f'cluster>{cluster_id}>storage_set[0]>name')
+            current_storage_set_data = Conf.get(
```

Comment:
Change variable from `current_storage_set_data` to `storage_set_info` and all references

Reply:
Updated